### ISSUE TITLE
Add Page Data Parsing Capability

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#14]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
There is a need to add page specific data parsing capability. Hence, let's add one and part it as `.Data.Page`. String data applies before transformation.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
Page level data file is made available.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
Page level data file is not available (at least not fully).

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 6a419de8c966a700a0c5a3e946b08ba4b701f017

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
